torchrecipes/audio/source_separation/module/conv_tasnet.py

```diff
@@ -0,0 +1,201 @@
+from dataclasses import dataclass, field
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Mapping,
+    List,
+    Optional,
+    Tuple,
+    Union,
+)
+
+import hydra
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+from torch.optim.lr_scheduler import _LRScheduler
+from torchaudio.models import ConvTasNet
+from torchrecipes.audio.source_separation.loss import si_sdr_loss
+from torchrecipes.audio.source_separation.metrics import sdri_metric, sisdri_metric
+from torchrecipes.core.conf import ModuleConf
+from torchrecipes.utils.config_utils import get_class_config_method, config_entry
+
+
+def _get_model(
+    num_sources=2,
+    enc_kernel_size=16,
+    enc_num_feats=512,
+    msk_kernel_size=3,
+    msk_num_feats=128,
+    msk_num_hidden_feats=512,
+    msk_num_layers=8,
+    msk_num_stacks=3,
+    msk_activate="relu",
```

Comment:
Do we want to make these options configurable? It seems that these are not exposed anywhere yet.

Reply:
I feel like the recipe is to reproduce the results from the original paper. Are users expected to change the model settings or training parameters in the recipe?

Reply:
I see. I think users might still want play around with the model and their own datasets. In that case, it'd be great if they can do so via the command line. Anyway, it's up to you. We can also do that later if needed.